src/main/java/frc/robot/subsystems/vision/Vision.java

```diff
@@ -53,10 +50,8 @@ public Vision(PhotonCamera cameraName, Transform3d robotToCamera) {
     this.photonPoseEstimator = photonPoseEstimator;
 
     // Initialize telemetry
-    runCountTelemetry = new CountPerPeriodTelemetry("Vision - " + cameraName.getName() + " - runs per s", 1);
-    setAtomicCountTelemetry = new CountPerPeriodTelemetry(
-        "Vision - " + cameraName.getName() + " - set atomic count per s",
-        1);
+    runCountTelemetry = new CountPerPeriodTelemetry("Vision/" + cameraName + " - runs per s", 1);
+    setAtomicCountTelemetry = new CountPerPeriodTelemetry("Vision/" + cameraName + " - meas per s push", 1);
```

Comment:
Could use `String.format()` here if you want; not required though

Reply:
I think I will create a simple helper function that returns this telemetry string as it is currently duplicated on 8+ places (vision/ + cameraname + telemetry name).